homeassistant/components/litejet/switch.py

```diff
@@ -43,56 +41,56 @@ def __init__(self, entry_id, lj, i, name):  # pylint: disable=invalid-name
         self._entry_id = entry_id
         self._lj = lj
         self._index = i
-        self._state = False
-        self._name = name
+        self._attr_is_on = False
+        self._attr_name = name
 
     async def async_added_to_hass(self) -> None:
         """Run when this Entity has been added to HA."""
         self._lj.on_switch_pressed(self._index, self._on_switch_pressed)
         self._lj.on_switch_released(self._index, self._on_switch_released)
+        self._lj.on_connected_changed(self._on_connected_changed)
 
     async def async_will_remove_from_hass(self) -> None:
         """Entity being removed from hass."""
         self._lj.unsubscribe(self._on_switch_pressed)
         self._lj.unsubscribe(self._on_switch_released)
+        self._lj.unsubscribe(self._on_connected_changed)
 
     def _on_switch_pressed(self):
-        _LOGGER.debug("Updating pressed for %s", self._name)
-        self._state = True
+        self._attr_is_on = True
         self.schedule_update_ha_state()
```

Comment:
Since `LiteJetSwitch` does not implement `async_update`, you can just call `self.async_write_ha_state()` which will update the state immediately. That can be implemented in a separate PR though.